MapboxCoreNavigation.podspec

```diff
@@ -40,7 +40,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxCoreNavigation"
 
-  s.dependency "MapboxDirections.swift", "~> 0.15"
+  s.dependency "MapboxDirections.swift", "~> 0.16.0"
```

Comment:
This was the problem: we were specifying only the minor version, so the tadpole operator would float until the next major version (1.0). By contrast, Carthage [intentionally violates semver](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#version-requirement) by being more strict pre-1.0.

Reply:
Maybe worth changing now?